app/app.js

```diff
@@ -969,19 +950,22 @@ var AppComponent = React.createClass({
     this.redirectToDefaultRoute();
     trackMetric('Signup Verified');
   },
-
   handleAcceptedTerms: function() {
-    this.setState({
-      showingAcceptTerms: false
+    var self = this;
+    var acceptedDate = sundial.utcDateString();
+    app.api.user.acceptTerms({ terms: acceptedDate },function(err) {
+      if(_.isEmpty(err)){
```

Comment:
yeah no reason - made sense at the time but makes no sense on a second look :) Fixed